evap/staff/templates/staff_evaluation_form.html

```diff
@@ -11,7 +11,16 @@ <h3>
     </h3>
     {% if evaluation_form.instance.id %}
         <div class="mb-2">
-            {% include 'course_badges.html' %}
+            {% if evaluation.state == 'new' %}
+                <span class="badge badge-light"><span class="fas fa-circle icon-yellow"></span> {% trans 'In preparation' %}</span>
+            {% elif evaluation.state == 'prepared' %}
+                <span class="badge badge-light"><span class="far fa-square icon-gray"></span> {% trans 'Awaiting editor review' %}</span>
+            {% elif evaluation.state == 'editor_approved' %}
+                <span class="badge badge-light"><span class="far fa-check-square icon-yellow"></span> {% trans 'Approved by editor' %}</span>
+            {% elif evaluation.state == 'approved' or evaluation.state == 'in_evaluation' or evaluation.state == 'evaluated' or evaluation.state == 'reviewed' or evaluation.state == 'published' %}
+                <span class="badge badge-light"><span class="far fa-check-square icon-green"></span> {% trans 'Approved by manager' %}</span>
+            {% endif %}
+            {% include 'course_badges.html' with course=evaluation.course %}
```

Comment:
there's the `statename` filter which produces different strings which are also shown to contributors. please check whether that should be unified.

Reply:
hm, good question. the strings used here are the ones from the staff semester page where we're not really using states as a concept for display in the ui anymore but the four categories (preparation, evaluation phase, text answer review, results publishing) in which the evaluations proceed.
i'm not sure and open for suggestions on general progression in this matter.

Reply:
one list for staff, one for contributors, or one for both if possible